mPowerSDK/Dashboard/APHLineGraphView.m

```diff
@@ -70,4 +103,138 @@ - (void)drawXAxis
     }
 }
 
+- (void)drawGraphForPlotIndex:(NSInteger)plotIndex {
+	[super drawGraphForPlotIndex:plotIndex];
+
+	if (self.shouldDrawLastPoint) {
+		[self drawLastPointDot:plotIndex];
+	}
+}
+
+- (void)drawLastPointDot:(NSInteger)plotIndex {
+	__block NSInteger i = -1;
+	[self.dataPoints enumerateObjectsUsingBlock:^(NSNumber *dataPoint, NSUInteger idx, BOOL *stop) {
```

Comment:
What is this suppose to be doing? if it is a reverse search to find the last point that is NOT NSNotFound then do a reverse enumeration.